### ISSUE TITLE
[ENHANCEMENTS] Additional question Exception to subfolder 

### ISSUE THREAD

**Issue by tomcatcw1980** [tobychui/zoraxy#341]
Hi There,

great project, many thanks for your work tobychui.

I'll pick up on this topic here again: #25 

I just want it the other way around: How should I set it up if I want to allow everything, but a basic auth query should appear for a certain subfolder? Ex: 

https://sub.foo.bar/* should be allowed, but
https://sub.foo.bar/admin the query should come up

Would be nice if you have an idea.

Many thanks and best regards

**Comment by ruben-conmag** [comment]
Hi, love the project ! And I'm also interested in this question. I have a similar situation

**Comment by tobychui** [comment]
@ruben-conmag For now you can only opt-out of a few certain paths but not opt-in for a given path. The config is under "Edit Credential" under Advance Settings

![Image](https://github.com/user-attachments/assets/11c67ecc-8365-4fb4-a56f-cf6b53758d88)


![Image](https://github.com/user-attachments/assets/95464652-f286-4eac-b98d-87db9d336c1b)

**Comment by ruben-conmag** [comment]
Hi ! Thanks a lot for the answer @tobychui ! But that is the current behaviour. I was asking if there is some advancement on the enhancement described in the issue.

**Comment by tobychui** [comment]
@ruben-conmag No, in fact basic auth shall not be used for complex credential / access management in the first place. Maybe I will implement this feature in Zoraxy SSO later if I got the time.
